src/screens/consul/ConsulIndexScreen.js

```diff
@@ -53,13 +53,13 @@ const showRegistrationFailAlert = (navigation) =>
 
 /* eslint-disable complexity */
 export const ConsulIndexScreen = ({ navigation, route }) => {
-  const [refreshing, setRefreshing] = useState(false);
-  const [sortingType, setSortingType] = useState(INITIAL_TOP_SORTING);
   const [filterType, setFilterType] = useState(INITIAL_TOP_FILTERING_FOR_POLLS);
   const [queryVariables, setQueryVariables] = useState(route.params?.queryVariables ?? {});
+  const [sortingType, setSortingType] = useState(INITIAL_TOP_SORTING);
   const bookmarkable = route.params?.bookmarkable;
-  const query = route.params?.query ?? '';
   const extraQuery = route.params?.extraQuery ?? '';
+  const query = route.params?.query ?? '';
+  let type = null;
```

Comment:
this `type` is used for two purposes, right? is there a specific reason using it globally? it is very not clear when reading `type = null`.
i saw the first for sorting list data, where `type` is used to store a selected one. couldn't it be a local `const selectedSortingType` for example?
and second usage is for filtering, where `type` is also used to store a selected one. couldn't it be a local `const selectedFilterType` for example?
this way there would no `type` variable, which has not much meaning and is used for multiple purposes.

Reply:
Good suggestion. I wanted to do 2 different jobs with a single variable.

Reply:
8d1121aeab4ff33f5e37bc5f2568faec55c0b11f